sage-ios/SAGE/SAGE/ProfileViewController.swift

```diff
@@ -0,0 +1,101 @@
+//  ProfileViewController.swift
+//  SAGE
+//
+//  Created by Erica Yin on 11/21/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class ProfileViewController: UITableViewController {
+
+    var user: User?
+    var profileView = ProfileView()
+    var activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView(activityIndicatorStyle: .Gray)
+    
+    init(user: User) {
+        self.user = user
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        self.user = User()
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setupHeader() {
+        self.tableView = UITableView(frame: self.tableView.frame, style: .Grouped)
+        self.tableView.tableHeaderView = self.profileView
+        let headerOffset = self.profileView.viewHeight + CGFloat(40)
+        var headerFrame = self.tableView.tableHeaderView!.frame
+        headerFrame.size.height = headerOffset
+        self.profileView.frame = headerFrame
+        self.tableView.tableHeaderView = self.profileView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupHeader()
+        self.tableView.tableFooterView = UIView()
+        
+        self.view.addSubview(self.activityIndicator)
+        self.activityIndicator.centerHorizontally()
+        self.activityIndicator.centerVertically()
+        self.activityIndicator.startAnimating()
+        
+        self.getUser()
+    }
+    
+    func getUser() {
+        ProfileOperations.getUser(self.user!, completion: { (user) -> Void in
+            self.profileView.setupWithUser(user)
+            self.activityIndicator.stopAnimating()
+            self.tableView.reloadData()
+            }) { (errorMessage) -> Void in
+                // display error
+        }
+    }
+        
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return 44.0
+    }
+    
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        if LoginOperations.getUser()!.id == self.user!.id {
+            return 2
+        }
+        return 0
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = UITableViewCell()
+        cell.selectionStyle = .None
+        cell.textLabel!.font = UIFont.normalFont
+        if indexPath.section == 0 {
+            cell.textLabel!.text = "All Check Ins"
+            cell.accessoryType = UITableViewCellAccessoryType.DisclosureIndicator
+        } else if indexPath.section == 1 {
+            cell.textLabel!.text = "Log Out"
+            cell.textLabel?.textColor = UIColor.redColor()
+            cell.textLabel!.textAlignment = NSTextAlignment.Center
```

Comment:
.Center works - if the compiler can tell what type of enum itll be, you dont have to include the enum name before the .